SonarRunner.Shim/PropertiesFileGenerator.cs

```diff
@@ -93,32 +96,38 @@ public static ProjectInfoAnalysisResult GenerateFile(AnalysisConfig config, ILog
         /// Loads SARIF reports from the given projects and attempts to fix
         /// improper escaping from Roslyn V1 (VS 2015 RTM) where appropriate.
         /// </summary>
-        private static void FixSarifReport(ILogger logger, IEnumerable<ProjectInfo> projects, IRoslynV1SarifFixer fixer /* for test */)
+        private static void FixSarifReports(ILogger logger, IEnumerable<ProjectInfo> projects, IRoslynV1SarifFixer fixer /* for test */)
         {
             // attempt to fix invalid project-level SARIF emitted by Roslyn 1.0 (VS 2015 RTM)
             foreach (ProjectInfo project in projects)
             {
-                Property reportPathProperty;
-                bool tryResult = project.TryGetAnalysisSetting(RoslynV1SarifFixer.ReportFilePropertyKey, out reportPathProperty);
-                if (tryResult)
+                FixSarifReport(logger, project, fixer, RoslynV1SarifFixer.CSharpLanguage, ReportFileCsharpPropertyKey);
+                FixSarifReport(logger, project, fixer, RoslynV1SarifFixer.VBNetLanguage, ReportFileVbnetPropertyKey);
```

Comment:
Just wondering: is there a way to know the language of the current project so that we can directly try to fix the correct file and avoid trying to fix blindly for both langages?